particles/Chooser/Chooser.js

```diff
@@ -146,7 +146,7 @@ ${productStyles}
       return {
         items: state.values.map(({rawData}, index) => {
           return Object.assign({
-            subId: rawData.name.replace(/ /g, '').toLowerCase(),
+            subId: state.values[index].id.replace(/[^a-zA-Z0-9]/g, '').toLowerCase(),
```

Comment:
Why not just encode the actual id?

Reply:
done:
1) the first parameter was rawData inside the values element, rearranged the code.
2) set subid to just id